.github/labeler.yml

```diff
@@ -0,0 +1,50 @@
+GHA:
```

Comment:
oh! missed this one; looks like we have `area/ci`; that's worth updating before merging

Reply:
Updated.